ios/UIViewController+RNScreens.m

```diff
@@ -0,0 +1,52 @@
+#import "UIViewController+RNScreens.h"
+#import "RNSScreenContainer.h"
+#import "RNSScreenStack.h"
+#import "RNSScreen.h"
+
+#import <objc/runtime.h>
+
+
+@implementation UIViewController (RNScreens)
+
+- (UIViewController *)reactNativeScreensChildViewControllerForStatusBarStyle
+{
+  UIViewController* lastViewController = [[self childViewControllers] lastObject];
+  if ([lastViewController isKindOfClass:[RNScreensNavigationController class]] || [lastViewController isKindOfClass:[RNScreensViewController class]] || [lastViewController isKindOfClass:[RNSScreen class]]) {
+    return lastViewController;
```

Comment:
we have all these swizzled method defined for RNScreensViewController and for RNScreenNavigationController. It seems to me that it'd be better to redirect the call to those implementations instead of having it implemented twice. I understand that what we want to do here is to forward calls to last child controller, but we only want to do it so long `self` is not one of the view controllers defined by screens library (RNScreensVC or RNScreenNavConroller)

Reply:
`RNScreenNavigationController` doesn't use the swizzled methods, probably due to `UINavigationController` having own implementations of them, that don't call `super`, so I am not sure if it would help with anything.

Reply:
How could then it be possible that `lastViewController` is of type `RNSScreen`? We should olways expect it to be of a container type (either RNSSVC or RNSNC)

Reply:
These methods can be used in `RNScreensViewController` too iirc.